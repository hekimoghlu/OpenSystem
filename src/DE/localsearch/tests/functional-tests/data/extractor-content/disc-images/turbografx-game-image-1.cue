CATALOG 0000000000000
FILE "foo.bin" BINARY
  TRACK 01 AUDIO
    INDEX 01 00:00:00
FILE "turbografx-game-image-1.bin" BINARY
  TRACK 02 MODE1/2352
    INDEX 00 00:00:00
    INDEX 01 00:03:00
