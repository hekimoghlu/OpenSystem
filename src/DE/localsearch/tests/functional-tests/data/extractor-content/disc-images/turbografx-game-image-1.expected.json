{
    "test": {
        "Filename": "turbografx-game-image-1.cue",
        "Comment": "Turbografx game image with CUE file"
    },
    "metadata": {
        "@graph": [
	    {
		"@type": [ "nfo:GameImage" ],
		"nie:mimeType": "application/x-cue"
	    }
	]
    }
}
